views/js/controller/creator/views/property.js

```diff
@@ -129,17 +129,31 @@ function($, ui, DataBinder, templates){
         * @private
         */
         function propValidation() {
-            var $togglers;
+            const $togglers = $('#saver');
+
             $view.on('validated.group', function(e, isValid){
                 if(e.namespace === 'group'){
-                    $togglers = $('#test-creator .property-toggler, #saver');
+                    let testSectionId = "#" + $('.tlb-button-on').parents('.section').attr("id");
                     if(isValid === true){
                         $togglers.removeClass('disabled');
+                        $(testSectionId).css("border-left", "solid 5px #a4bbc5");
                     } else {
+                        $(testSectionId).css("border-left", "solid 5px #ba122b");
                         $togglers.addClass('disabled');
                     }
+
+            //disables save button if span.validate-error is present in any property input
+            $togglers.on('mouseenter', function(){
```

Comment:
can we check enabling Save button in listener  `$view.on('validated.group', function(e, isValid){` because listening `mouseenter` is wrong way